samples/uss-profile-sample/package.json

```diff
@@ -46,7 +46,7 @@
     "typescript": "^5.1.3"
   },
   "peerDependencies": {
-    "@zowe/zos-files-for-zowe-sdk": ">=8.0.0-next.0 <8.0.0",
-    "@zowe/zos-uss-for-zowe-sdk": ">=8.0.0-next.0 <8.0.0"
+    "@zowe/zos-files-for-zowe-sdk": ">=8.0.0 <9.0.0",
+    "@zowe/zos-uss-for-zowe-sdk": ">=8.0.0 <9.0.0"
```

Comment:
It should be safe to use `^8.0.0` here 😋

Reply:
would `^8.0.0` only cover v8 patches and require an update with 8.1.0 where `>=8.0.0` will cover any 8.x release?

Reply:
I was wondering about adding ^8.0.0 to actual dependencies though to get the patch fixes automatically, what do you all think?

Reply:
updates in commit https://github.com/zowe/zowe-explorer-vscode/pull/3146/commits/a3c3c1cd977a703224ca58636e1f81fa424a2664